src/simularium/types.ts

```diff
@@ -154,7 +154,6 @@ export interface FileReturn {
 
 // IMPORTANT: Order of this array needs to perfectly match the incoming data.
 export const AGENT_OBJECT_KEYS = [
-    // TODO: convert "vis-type" to visType at parse time
```

Comment:
I feel like this key still needs some sort of comment here about how this is for json data coming from the server?  or maybe not.  Does octopus send json data out?